### DISCUSSION TITLE
Possibility to show Entries containing a certain asset

### DISCUSSION THREAD

**Discussion by mnlmaier** [statamic/cms#3929] [General]
Hi guys,

for a client, it would be really helpful to display the Pages/Entries which use a certain asset, ideally directly in the asset view. From my perspective, this should be possible relatively easily (correct me if I am wrong) – let me know if this would be something worth adding in the Core Codebase or rather be shipped as a Package. I will be happy to help out :)

All the best and thanks for your great work!

**Comment by edalzell** [comment]
Sounds like something that wouldn't be in core, to be honest, so if you have something working, feel free to release it.
